src/Demos/12-norvig's-spelling-corrector.fsx

```diff
@@ -58,7 +58,7 @@ let files = downloadJob.AwaitResult()
 
 let fileSizesJob = 
     files
-    |> Distributed.map CloudFile.GetSize
+    |> DivideAndConquer.map CloudFile.GetSize
```

Comment:
@eiriktsarpalis Is the version of MBrace expected by this repo being kept in lock-step with the version installed by Brisk?